api/v1alpha3/azurecluster_webhook_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestAzureCluster_ValidateUpdate(t *testing.T) {
+	g := NewWithT(t)
+
+	tests := []struct {
+		name    string
+		cluster *AzureCluster
+		updated *AzureCluster
+		wantErr bool
+	}{
+		{
+			name:    "no change",
+			cluster: createAzureCluster(t, "westus2"),
+			updated: createAzureCluster(t, "westus2"),
+			wantErr: false,
+		},
+		{
+			name:    "no change",
+			cluster: createAzureCluster(t, "westus2"),
+			updated: createAzureCluster(t, "eastus"),
+			wantErr: true,
+		},
+	}
+	for _, tc := range tests {
+		tc := tc
```

Comment:
I don't think this is needed

Reply:
You are correct, it is not needed because `t.Run` is running synchronously. I usually advocate for this style as it often missed when someone goes to make a test parallel and then runs into the pernicious issue of closure capture.

https://play.golang.org/p/GGrFU73Na8S

Reply:
Added the parallel invocation to settle the debate :)